doc/nrf/security/ap_protect.rst

```diff
@@ -170,43 +170,144 @@ Based on the available implementation types, you can configure the access port p
+
+Secure AP-Protect
+=================
+
+With :ref:`Trusted Firmware-M (TF-M) <ug_tfm>` comes :ref:`security by separation <app_boards_spe_nspe>`, enabling a Secure Processing Environment (SPE) that is isolated from the Non-Secure Processing Environment (NSPE).
+TF-M is available for the nRF53 and nRF91 Series devices.
+
+While AP-Protect blocks access to all CPU registers and memories, Secure AP-Protect limits the CPU access to the non-secure side only.
+This allows debugging of the NSPE, while the SPE debugging is blocked.
+
+The following Kconfig options for enabling Secure AP-Protect are available for the nRF91x1 and nRF53 Series devices:
+
+* :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK`
+* :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING`
+* :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USE_UICR`
+
+In addition, you can enable hardware Secure AP-Protect by setting the ``UICR.SECUREAPPROTECT`` register.
+
+Enabling software Secure AP-Protect with :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK`
+-------------------------------------------------------------------------------------------
 
-To enable only the Secure AP-Protect, run the following command:
+Setting the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK` Kconfig option to ``y`` and compiling the firmware enables the secure access protection mechanism for SoCs of the nRF53 Series.
+
+Enabling this Kconfig option writes the secure debugger register in the ``SystemInit()`` function to lock the secure access port protection at every boot.
+In addition to this, the ``UICR.SECUREAPPROTECT`` register should be written as instructed in :ref:`app_secure_approtect_uicr_approtect`.
+
+.. note::
+    For multi-image builds, this Kconfig option needs to be set for the first image (usually a bootloader).
+    Otherwise, the software Secure AP-Protect will not be sufficient as the debugger can be attached to the SPE after the first image opens the software Secure AP-Protect with the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USE_UICR` Kconfig option, which is the default value.
+
+    When using sysbuild, set the sysbuild Kconfig option ``SB_CONFIG_SECURE_APPROTECT_LOCK``, which enables the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK` Kconfig option for all images.
+
+.. important::
+    On the nRF91x1 Series devices, the register setting related to the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK` Kconfig option does not persist in System ON IDLE mode.
+    You must lock the ``UICR.SECUREAPPROTECT`` register to enable the hardware Secure AP-Protect mechanism as instructed in :ref:`app_secure_approtect_uicr_approtect`.
+
+Enabling software Secure AP-Protect with :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING`
+----------------------------------------------------------------------------------------------------
+
+Setting the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USE_UICR` Kconfig option to ``y`` disables the software Secure AP-Protect mechanism.
```

Comment:
```suggestion
Setting the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING` Kconfig option to ``y`` disables the software Secure AP-Protect mechanism.
```

Reply:
Well, this was a copy-paste issue, good catch.